lxqt-admin-time/timezone.cpp

```diff
@@ -47,40 +47,41 @@ Timezone::Timezone(const QStringList & zones, const QString & currentimezone, QW
     reload();
 }
 
-Timezone::~Timezone()
+TimezonePage::~TimezonePage()
 {
     delete ui;
 }
 
-void Timezone::reload()
+void TimezonePage::reload()
 {
     ui->list_zones->setCurrentRow(-1);
     mZoneChanged = false;
     QList<QListWidgetItem *> list = ui->list_zones->findItems(ui->label_timezone->text(),Qt::MatchExactly);
     if (list.count())
         ui->list_zones->setCurrentItem(list.at(0));
 
-    emit changed(mZoneChanged);
+    emit changed();
 }
 
-QString Timezone::timezone() const
+QString TimezonePage::timezone() const
 {
     if (ui->list_zones->currentItem())
         return ui->list_zones->currentItem()->text();
     else
         return QString();
 }
 
-void Timezone::on_list_zones_itemActivated(QListWidgetItem * item)
+void TimezonePage::on_list_zones_itemSelectionChanged()
```

Comment:
Just wanted to drop a note, why the timezone isn't applied ... but you're quicker :)